workers/unity/Packages/com.improbable.gdk.core/Systems/ComponentUpdateSystem.cs

```diff
@@ -81,45 +73,17 @@ public List<EntityId> GetComponentsRemoved(uint componentId)
 
         public Authority GetAuthority(EntityId entityId, uint componentId)
         {
-            if (!componentIdToManager.TryGetValue(componentId, out var manager))
-            {
-                throw new ArgumentException("Component ID not recognized");
-            }
-
-            return ((IAuthorityManager) manager).GetAuthority(entityId);
+            return worker.View.GetAuthority(entityId, componentId);
         }
 
         public void AcknowledgeAuthorityLoss(EntityId entityId, uint componentId)
         {
             worker.MessagesToSend.AcknowledgeAuthorityLoss(entityId.Id, componentId);
         }
 
-        internal ComponentType[] GetInitialComponentsToAdd(uint componentId)
-        {
-            if (!componentIdToManager.TryGetValue(componentId, out var manager))
-            {
-                throw new ArgumentException("Component ID not recognized");
-            }
-
-            return manager.GetInitialComponents();
-        }
-
         public bool HasComponent(uint componentId, EntityId entityId)
```

Comment:
Do these APIs make sense here anymore?

Reply:
This one and `GetAuthority`

Reply:
Discussed offline. Yes for the moment but it's an ongoing task to decide if this will remain the case.